src/osemgrep/language_server/scan_helpers/Scan_helpers.ml

```diff
@@ -157,7 +157,7 @@ let run_semgrep ?(targets : Fpath.t list option) ?rules ?git_ref
    on django) before the notifications are received by the extension.
    In the interim, we will just continue to hook into core.
 *)
-let run_core_search rule (file : Fpath.t) =
+let run_core_search xconf rule (file : Fpath.t) =
```

Comment:
curious if we can do something to speed up the rest of the LS scanning similar to here. But that's future work